app/frontend/src/components/Submit.js

```diff
@@ -13,15 +14,33 @@ export const Submit = () => {
 	})
 
 	const submit = async () => {
+		if (data.reference && data.reference.edit) {
+			console.log(
+				'Please save changes made in reference before submitting'
+			)
+			return
+		}
+
+		if (data.structured_names.length === 0 || data.relations.length === 0) {
```

Comment:
Having no added structured names is probably a valid scenario if all new relations are between existing names.